packages/identifier-resolution/src/functions/externalOIDFIdentifier.ts

```diff
@@ -9,6 +9,7 @@ import { IAgentContext } from '@veramo/core'
 import { IOIDFClient } from '@sphereon/ssi-sdk.oidf-client'
 import { contextHasPlugin } from '@sphereon/ssi-sdk.agent-config'
 import { IJwsValidationResult } from '../types/IJwtService'
+import {decodeBase64url} from "@veramo/utils";
```

Comment:
Do not include veramo utils. It deps on almost everything in veramo.

We have functions for base64 url decoding. And otherwise it is as simple as u8a.fromString(input, 'base64url')

Reply:
Replaced with uint8arrays as suggested